doc/reference/reference_lua/box_info.rst

```diff
@@ -280,7 +280,7 @@ variables.
       vector clock table.
       Also the replica sends its entire vector clock table in response
       to a master's heartbeat message, see the heartbeat-message examples
-      in section :ref:`Binary protocol -- replication <box_protocol-replication>`.
+      in section :ref:`Binary protocol -- replication <box_protocol-heartbeat>`.
```

Comment:
are you sure about the link label?

Reply:
yes, I've added this tag [here](https://github.com/tarantool/doc/pull/1366/files#diff-90b504e032fff8895ef9e66d545c19f2R1013)